### ISSUE TITLE
Elements must have sufficient color contrast - QIT About Link

### ISSUE THREAD

**Issue by davefollett** [codingblocks/podcast-app#215]
## 👣 Steps to reproduce
[Run the aXe Accessibility Test](https://github.com/codingblocks/podcast-app/wiki/Accessibility-(a11y)-Testing#run-axe-accessibility-test) on the main page of the site: https://qit.cloud or http://localhost:3000

## ⚛️ Element(s) with issue
`<span class="sc-EHOje jvpdfr" data-nav="about" href="/about/">qit</span>`

## 💥 Impact rating
serious

## 🛠️ Recommended fix
Element has insufficient color contrast of 3.77 (foreground color: #ffffff, background color: #359189, font size: 16.8pt, font weight: normal). Expected contrast ratio of 4.5:1

## 📚 Reference information
https://dequeuniversity.com/rules/axe/3.0/color-contrast?application=AxeChrome

**Comment by davefollett** [comment]
According to the [ Tanaguru Contrast-Finder](http://contrast-finder.tanaguru.com/result.html?foreground=%23ffffff&background=%23359189&ratio=4.5&isBackgroundTested=false&algo=Rgb) we should switch the text color in the header from white to black if we want to keep the green/blue background color.

If we want to keep the white text, [here](http://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%23359189&ratio=4.5&isBackgroundTested=true&algo=Rgb&distanceSort=asc ) are some recommended background color changes.
